app/routes/session.py

```diff
@@ -178,7 +182,20 @@ def get_sign_out():
 
 @session_blueprint.route("/signed-out", methods=["GET"])
 def get_signed_out():
-    return render_template(template="signed-out")
+    if not cookie_session:
+        return redirect(url_for("session.get_session_expired"))
+
+    survey_type = get_survey_type()
+    survey_config = get_survey_config(theme=survey_type)
+    redirect_url = (
+        survey_config.account_service_todo_url
+        if survey_type is SurveyType.BUSINESS
+        else survey_config.account_service_log_out_url
+    )
```

Comment:
Actually, this doesn't work because we have other survey types that need to go to todo url, so I would just say go to todo if we have one. Social/Health won't have one.
```suggestion
    redirect_url = (
        survey_config.account_service_todo_url
        or survey_config.account_service_log_out_url
    )
```